bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastHandler.java

```diff
@@ -336,7 +336,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                                 if ("none".equals(localRole)) {
                                     json = "{\"group_id\":\"" + groupId + "\", \"zone\":\"" + mclinkSetupZone
                                             + "\", \"type\":\"add\", \"client_list\":[\"" + this.host + "\"]}";
-                                    logger.debug("setServerInfo json: {}", json);
+                                    logger.trace("setServerInfo json: {}", json);
```

Comment:
Please don't dump json data into the logs in the future. You could use the debugger during development or add these dumps to dedicated JARs for certain users if they encounter some problems.